### PR TITLE
add gain & elevation to valid beacon and witness reports

### PR DESCRIPTION
**Pull request by andymck**

### CODE REVIEW

**Review by bbalser — APPROVED** [review]

**Review by jeffgrunewald — APPROVED** [review]

File: src/service/poc_lora.proto [hunk 1]
```diff
@@ -156,6 +162,12 @@ message lora_verified_witness_report_v1 {
   // the participant to which the reason applies,
   // which rendered the report as invalid
   invalid_participant_side participant_side = 8;
+  /// the transmit gain value of the gateway in dbi x 10
+  /// For example 1 dbi = 10, 15 dbi = 150
+  /// derived from gateway metadata
+  int32 gain = 9;
+  /// The asserted elevation of the gateway, derived from gateway metadata
+  int32 elevation = 10;
```
Comment (jeffgrunewald) [inline src/service/poc_lora.proto:170]: shouldn't this be in the `lora_valid_witness_report`? how does a "verified" witness relate to a "valid" witness?

File: src/service/poc_lora.proto [hunk 2]
```diff
@@ -106,6 +106,12 @@ message lora_valid_beacon_report_v1 {
   // integer representation of a 4-point precision decimal multiplier
   // based on the number of witnesses to a poc event
   uint32 reward_unit = 5;
+  /// the transmit gain value of the gateway in dbi x 10
+  /// For example 1 dbi = 10, 15 dbi = 150
+  /// derived from gateway metadata
+  int32 gain = 6;
+  /// The asserted elevation of the gateway, derived from gateway metadata
+  int32 elevation = 7;
```
Comment (madninja) [inline src/service/poc_lora.proto:114]: Declare units relative to what

File: src/service/poc_lora.proto (hunk 1 above)
Comment (jeffgrunewald) [reply src/service/poc_lora.proto:170]: confirmed the valid report for witnesses is deprecated

File: src/service/poc_lora.proto (hunk 2 above)
Comment (andymck) [reply src/service/poc_lora.proto:114]: unclear to me what units elevation is in, i imagine its meters but its not defined in any source proto that I am aware off.  Confirm here if you know the units

**Review by jeffgrunewald — APPROVED** [review]

File: src/service/poc_lora.proto (hunk 2 above)
Comment (vihu) [reply src/service/poc_lora.proto:114]: I'm pretty sure that elevation is AGL (above ground level) in meters.

**Review by vihu — APPROVED** [review]

**Pull request merged**